libs/model-ad/home/src/lib/home.component.html

```diff
@@ -1,4 +1,5 @@
-Model-AD home
+<h1>Model-AD Home</h1>
+<h2>based on Agora</h2>
```

Comment:
We aren't going to want to release with this on the home page. If you were able to verify the font in your dev container, and don't forsee it going randomly MIA in our environments somehow, we could just remove it now.

Reply:
@hallieswan is rewriting the home component